sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1 @@
+select "filmId", "title" from films where "rentalRate" >= 1 limit 50; 
```

Comment:
we want the cheap ones, less than a dollar

Reply:
well...ops